src/DefaultTestReporter.js

```diff
@@ -7,15 +7,18 @@
  */
 'use strict';
 
-const colors = require('./lib/colors');
+const chalk = require('chalk');
 const formatFailureMessage = require('./lib/utils').formatFailureMessage;
-const formatMsg = require('./lib/utils').formatMsg;
 const path = require('path');
 const VerboseLogger = require('./lib/testLogger');
 
-const FAIL_COLOR = colors.RESET + colors.RED_BG + colors.BOLD;
-const PASS_COLOR = colors.RESET + colors.GREEN_BG + colors.BOLD;
```

Comment:
Checked, it doesn't. I haven't tracked it down for sure but it looks like this is most likely due to 2 things trying to write to stdout at the same time. It only happens when `console.error` is called in the tests themselves and then another test finishes. So in the middle of printing out the `console.error` (which we color red), we start printing out the PASS line. With the fix you have now, our PASS line is fixed but then the rest of the error is printed normally (in plain white text for me). I *think* this is happening because we create a new `console` instance when creating each jsdom environment, instead of somehow using the same one.